test/fixtures/hdfs-fixture/src/main/java/hdfs/MiniHDFS.java

```diff
@@ -91,6 +94,8 @@ public static void main(String[] args) throws Exception {
             cfg.set(DFSConfigKeys.DFS_NAMENODE_ACLS_ENABLED_KEY, "true");
             cfg.set(DFSConfigKeys.DFS_BLOCK_ACCESS_TOKEN_ENABLE_KEY, "true");
             cfg.set(DFSConfigKeys.IGNORE_SECURE_PORTS_FOR_TESTING_KEY, "true");
+            cfg.set(HdfsClientConfigKeys.DFS_DATA_TRANSFER_PROTECTION_KEY, "authentication");
```

Comment:
is this still needed if we have `dfs.encrypt.data.transfer` set to true? According to the docs, one should supersede the other, see https://hadoop.apache.org/docs/r2.6.0/hadoop-project-dist/hadoop-hdfs/hdfs-default.xml